website/docs/r/cos_bucket.html.markdown

```diff
@@ -158,6 +178,7 @@ The following arguments are supported:
 * `endpoint_type` - (Optional, string) The type of the endpoint (public or private) to be used for buckets. Default value is `public`.
 
 * `force_delete` - (Optional, bool) Since Default value set to True, it will delete all the objects in the COS Bucket and then delete the bucket.  Default value is `true`.
+* **Note** - force_delete will timeout on buckets with a large amount of objects.  24 hours before you delete the bucket you can set an expire rule to remove all files over a day old.  
```

Comment:
I would think that you would want to indent this note so that it would appear under the `force_delete` option.

Also put `force_delete` in code backticks